### PR TITLE
Refactoring and Travis cleanup

### PR DESCRIPTION
**Pull request by cirelli94**

### CODE REVIEW

File: logger/logger_logrus.go [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-package utils
+// +build !rhel5
+
+package logger
```
Comment (amreo) [inline logger/logger_logrus.go:3]: The license header is missing here

File: logger/logger_rhel5.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
+// +build rhel5
+
```
Comment (amreo) [inline logger/logger_rhel5.go:2]: The license header is missing

File: logger/logger_logrus.go (hunk 1 above)
Comment (cirelli94) [reply logger/logger_logrus.go:3]: Thanks!

File: logger/logger_rhel5.go (hunk 2 above)
Comment (cirelli94) [reply logger/logger_rhel5.go:2]: Added thanks!

**Pull request closed without merging**